app/src/main/java/uk/ac/ebi/atlas/search/metadata/CellTypeWheelDao.java

```diff
@@ -82,10 +82,39 @@ public CellTypeWheelDao(SolrCloudCollectionProxyFactory proxyFactory) {
     }
 
     public ImmutableList<ImmutableList<String>> facetSearchCtwFields(String searchTerm, String species) {
-        // Build the facet in the query above:
-        // "facet": {
-        //   ...
-        // }
+        var queryBuilder = buildSolrQueryForCellTypeWheel(searchTerm);
```

Comment:
There are couple of duplications in these 2 methods (`facetSearchCtwFields` and `speciesSearchCtwFields`).
Could you please extract them to private methods?
Thanks